### DISCUSSION TITLE
Account Suspended and Immediately Reinstated – Request for Clarification

### DISCUSSION THREAD

**Discussion by Danish451** [community/community#140705] [Other Feature Feedback, Questions, & Ideas] [answered]
_Hello,

Ticket ID: 3036057
Email ID: [Redacted by moderator due to PII]

I am writing on behalf of my friend, Vishal Rawat (GitHub ID: [Vicky2114](https://github.com/Vicky2114)), who recently experienced an issue where his GitHub account was suspended without any clear reason, and then it was reinstated shortly after.

Vishal is unsure why this suspension occurred, as he hasn't violated any terms or policies to his knowledge. We would greatly appreciate any clarification on the cause of the suspension to ensure his account remains in good standing.

Please let us know if any further action is required to prevent this from happening again.

Thank you for your time and support._ @zackfern @martinwoodward @davidstaheli @Bestra @dmilburn

**Best regards,
Vishal's friend,
Danish Khan**

**Comment by Manavmodi4703** [comment]
Help him to get his repo back ASAP.

**Comment by ishika3838** [comment]
Dear GitHub Support Team,

I hope this message finds you well.

I am writing on behalf of my friend, Vishal Rawat (GitHub Username: Vicky2114, Email: vr6994056@gmail.com), who recently faced an unexpected suspension of his GitHub account. The account was subsequently reinstated; however, Vishal is unclear about the reasons behind this action.

He has reviewed GitHub's policies and believes he has not violated any terms. We kindly request your assistance in providing clarity on the cause of the suspension to ensure his account's compliance moving forward.

Additionally, if there are any steps Vishal should take to prevent a recurrence, please let us know.

Thank you for your time and support._ @zackfern @martinwoodward @davidstaheli @Bestra @dmilburn

Best regards,
Ishika Patni

**Comment by TestingAccount89** [comment]
Policy violations, fraudulent activities, abusive behavior, and security breaches... GitHub accounts can be suspended for violating terms of service, community guidelines, or engaging in malicious activities such as spamming, intellectual property infringement, or fraud, while repeated violations or persistent disruptive behavior can lead to permanent suspension.

Recently posting external links was deemed to be the reason for account ban. As simple as sharing google sheet links, web links etc counted as spamming.

**Reply by TestingAccount89** [reply]
Whoever put a thumbs down needs to know that people who posted external links, their account was recently banned. Try for yourself

**Reply by Vishal-2114** [reply]
okay so the reason is for sending any reference link in readme file only

**Reply by Vishal-2114** [reply]
can u please refer me any violation policies in readme file of external links added

**Reply by Danish451** [reply]
Please provide the proof where this all external links stuffs mentioned

**Reply by TestingAccount89** [reply]
> can u please refer me any violation policies in readme file of external links added

You are right, it is not a violation. But the Github thinks as such. Some coworkers faced this issue.

**Reply by TestingAccount89** [reply]
> Please provide the proof where this all external links stuffs mentioned

There is no proof, it is said by people who witnessed the recent incidents

**Reply by TestingAccount89** [reply]
> okay so the reason is for sending any reference link in readme file only

The people who were involved in discussions comments who posted links faced this. Also accounts without 2FA, newly created ones, etc

**Reply by Vishal-2114** [reply]
> > okay so the reason is for sending any reference link in readme file only
> 
> The people who were involved in discussions comments who posted links faced this. Also accounts without 2FA, newly created ones, etc

I've already enabled two-factor authentication (2FA) on this account, and it's been in use for a time now.

**Comment by Manavmodi4703** [comment]
Dear GitHub Support Team,

I hope this message finds you well.

I am writing on behalf of my friend, Vishal Rawat (GitHub Username: Vicky2114, Email: [vr6994056@gmail.com](mailto:vr6994056@gmail.com)), who recently faced an unexpected suspension of his GitHub account. The account was subsequently reinstated; however, Vishal is unclear about the reasons behind this action.

He has reviewed GitHub's policies and believes he has not violated any terms. We kindly request your assistance in providing clarity on the cause of the suspension to ensure his account's compliance moving forward.

Additionally, if there are any steps Vishal should take to prevent a recurrence, please let us know.

Thank you for your time and support._ @zackfern @martinwoodward @davidstaheli @Bestra @dmilburn

Best regards,
Manav Modi

**Comment by Vishal-2114** [comment]
Dear GitHub Support Team,

I hope this message finds you well.

I am Vishal Rawat (GitHub Username: Vicky2114, Email: vr6994056@gmail.com), and I recently faced an unexpected suspension of my GitHub account. While the account has been reinstated, I am unsure about the reasons behind the suspension, as I did not receive any prior warning or notification.

I have reviewed GitHub's terms and policies and believe I have not violated any of them. I would appreciate your help in clarifying the reason for the suspension to ensure compliance with GitHub's policies moving forward.

Additionally, I am encountering an issue with the password recovery process. Despite trying to reset my password, I have not received the reset email. This is causing a delay in accessing my account, which is essential for my daily personal projects and organizational work.

The ticket ID for this issue is 3036057. I would be grateful for any assistance you can provide to resolve this matter and prevent future occurrences.

Thank you for your support.  @zackfern @martinwoodward  @davidstaheli  @Bestra  @dmilburn

Best regards,
Vishal Rawat

**Comment by KumarTusharSahu** [comment]
Dear GitHub Support Team,

I hope this message finds you well.

I am writing on behalf of my friend, Vishal Rawat (GitHub Username: Vicky2114, Email: [vr6994056@gmail.com](mailto:vr6994056@gmail.com)), who recently faced an unexpected suspension of his GitHub account. The account was subsequently reinstated; however, Vishal is unclear about the reasons behind this action.

He has reviewed GitHub's policies and believes he has not violated any terms. We kindly request your assistance in providing clarity on the cause of the suspension to ensure his account's compliance moving forward.

Additionally, if there are any steps Vishal should take to prevent a recurrence, please let us know.

Thank you for your time and support._ @zackfern @martinwoodward @davidstaheli @Bestra @dmilburn

Best regards,
Tushar Kumar Sahu

**Comment by Vishal-2114** [comment]
Dear GitHub Support ,

I’m experiencing an issue similar to the one discussed in this GitHub Community thread: [GitHub Community Discussion](https://github.com/orgs/community/discussions/116569#discussioncomment-8990131).

Could you please review and help resolve this issue for my account?

Details:

Ticket ID: 3036057
GitHub Username: Vicky2114
Email: vr6994056@gmail.com
Additionally, I would like to request daily updates for the projects under my company’s organization as well as my college projects.

Thank you for your assistance, and I look forward to your prompt response @github-staff , @Github. 

Best regards,
Vishal Rawat

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Danish451 ! The best place for your friend to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). 

The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

Please note, contacting any member of GitHub Staff via unsolicited mentions or pings is strongly discouraged and may be considered a violation of our prohibition against harassment. Please review our [Code of Conduct](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct) for more information.

Let's work together to keep GitHub Community a place where people feel safe to participate by being respectful of them and their time.

Any repeat and future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
